gcc/cp/cp-tree.h

```diff
@@ -9001,6 +9007,31 @@ set_contract_const (tree t, bool constify)
   TREE_LANG_FLAG_4 (CONTRACT_CHECK (t)) = constify;
 }
 
+/* Test if EXP is a contract const wrapper node.  */
+
+inline bool
+contract_const_wrapper_p (const_tree exp)
```

Comment:
So, now, let's name this one contract_constify_expr_p.

Reply:
Uh, and then we don't need that macro either. Let's just use the function and nuke that macro too.